### PR TITLE
[l10n] Improve Portuguese (pt-BR) locale

### PR DESCRIPTION
**Pull request by cnHealth**
- [ ] I have followed (at least) the [PR section of the contributing guide](https://github.com/mui/mui-x/blob/HEAD/CONTRIBUTING.md#sending-a-pull-request).

### CODE REVIEW

File: packages/x-date-pickers/src/locales/ptBR.ts [hunk 1]
```diff
@@ -25,10 +25,10 @@ const ptBRPickers: Partial<PickersLocaleText<any>> = {
...
   start: 'Início',
   end: 'Fim',
-  // startDate: 'Start date',
-  // startTime: 'Start time',
-  // endDate: 'End date',
-  // endTime: 'End time',
+  startDate: 'Data de início',
+  startTime: 'Hora de início',
+  endDate: 'Data de Término',
+  endTime: 'Hoora de Término',
```
Comment (renangasperi) [inline packages/x-date-pickers/src/locales/ptBR.ts:31]: Hora de Término*

File: packages/x-date-pickers/src/locales/ptBR.ts [hunk 2]
```diff
@@ -84,17 +84,17 @@ const ptBRPickers: Partial<PickersLocaleText<any>> = {
...
-  // minutes: 'Minutes',
-  // seconds: 'Seconds',
-  // meridiem: 'Meridiem',
+   year: 'Ano',
+   month: 'Mês',
+   day: 'Dia',
+   weekDay: 'Dia da Semana',
+   hours: 'Horas',
+   minutes: 'Minutos',
+   seconds: 'Segundoos',
```
Comment (renangasperi) [inline packages/x-date-pickers/src/locales/ptBR.ts:93]: Segundos*

File: packages/x-date-pickers/src/locales/ptBR.ts (hunk 1 above)
Comment (LukasTy) [reply packages/x-date-pickers/src/locales/ptBR.ts:31]:
```suggestion
  endTime: 'Hora de Término',
```

File: packages/x-date-pickers/src/locales/ptBR.ts (hunk 2 above)
Comment (LukasTy) [reply packages/x-date-pickers/src/locales/ptBR.ts:93]:
```suggestion
   seconds: 'Segundos',
```

**Pull request merged**